### ISSUE TITLE
UI display of qualified predicates in results

### ISSUE THREAD

**Issue by karafecho** [RobokopU24/Issues#202]
This issue is to consider a better way to display qualified predicates in the UI. At present, a user has to know to first click on a predicate and then understand how to read the qualified predicate in the pop-up box. I don't have any suggestions, and perhaps documentation is sufficient, but this issue confuses users, especially because the qualified predicates are an exception, not the norm.

See https://github.com/NCATSTranslator/Evidence-Provenance-Confidence-Working-Group/blob/master/qualifier_example_data_and_queries.md for examples.

**Comment by cbizon** [comment]
I don't think that these are even being displayed now.  I think that this needs to be a pretty high priority when we work on the results.
